### PR TITLE
feat(storefront): strf-9174 Product object in quick view on localhost not consistent with prod:

### PR DESCRIPTION
**Pull request by bc-max**
#### What?

Quick-view in stencil-cli mode didn't include `reviews` sections if any.

#### Tickets / Documentation

-   [STRF-9174](https://bigcommercecloud.atlassian.net/browse/STRF-9174)

#### Screenshots (if appropriate)

image

cc @bigcommerce/storefront-team

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -33,8 +33,6 @@ function buildManifest(srcManifest, options) {
     pluginsByName['./plugins/renderer/renderer.module'].storeUrl = storeUrl;
     pluginsByName['./plugins/renderer/renderer.module'].storeSettingsLocale =
         options.storeSettingsLocale;
-    pluginsByName['./plugins/renderer/renderer.module'].customLayouts =
```
Comment (bc-max) [inline server/index.js:36]: this is duplicate of line 30 above

File: package-lock.json [hunk 2]
```diff
@@ -533,7 +533,7 @@
       "resolved": "https://registry.npmjs.org/@bigcommerce/stencil-styles/-/stencil-styles-3.0.0.tgz",
       "integrity": "sha512-5w2YpZxs01/Y4O+q6/kfIBSLGv0nirofvXayI3P9rcH5fWfgR1ljnBH033zisFU95QgpsOxKiLYvUHDc0vXiKg==",
       "requires": {
-        "@bigcommerce/node-sass": "git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
+        "@bigcommerce/node-sass": "@bigcommerce/node-sass@git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
```
Comment (jairo-bc) [inline package-lock.json:536]: revert please this :)

File: server/plugins/renderer/renderer.module.js [hunk 3]
```diff
@@ -457,7 +457,9 @@ internals.buildReqHeaders = ({
         'stencil-custom-templates': templates,
     };
 
-    if (!request.headers['stencil-config'] && stencilConfig) {
+    const config = request.headers['stencil-config'];
+
+    if ((!config || config === '{}') && stencilConfig) {
```
Comment (jairo-bc) [inline server/plugins/renderer/renderer.module.js:462]: So the header can be undefined and can be empty object?

Comment (bc-max) [reply server/plugins/renderer/renderer.module.js:462]: Not sure about undefined, I decided not to touch existing check just not to break anything. However, it definitely could be empty object. I saw that during debugging.

File: package-lock.json (hunk 2 above)
Comment (bc-max) [reply package-lock.json]: done

File: package-lock.json [hunk 4]
```diff
@@ -544,7 +544,7 @@
       "dependencies": {
         "@bigcommerce/node-sass": {
           "version": "git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
-          "from": "git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
+          "from": "@bigcommerce/node-sass@git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
```
Comment (bc-max) [inline package-lock.json:547]: @jairo-bc do I need to revert this as well?

Comment (jairo-bc) [reply package-lock.json:547]: yes please

Comment (bc-max) [reply package-lock.json]: done

**Review by jairo-bc — APPROVED** [review]

**Pull request merged**